### DISCUSSION TITLE
Problems getting window size in Bevy 0.10

### DISCUSSION THREAD

**Discussion by ghost** [bevyengine/bevy#8203] [Q&A] [answered]
I was following the tutorial https://mbuffett.com/posts/bevy-snake-tutorial/ in bevy 0.10
I have translated everything so far but I am finding unable to find the new form of Res<Window>
I am getting the following error: 
```
  the trait `bevy::prelude::Resource` is not implemented for `bevy::prelude::Window`
```

**Comment by hymm** [comment, marked as answer]
Windows are entities now.  You probably want something like `Query<&Window, With<PrimaryWindow>>`

**Reply by ghost** [reply]
Dankeschön
